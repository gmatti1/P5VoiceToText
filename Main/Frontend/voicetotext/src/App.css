.App-bg {
  margin: 0;
  position: absolute;
  background-image: url(medical-wallpaper.jpg);
  opacity:0.95;
  width: 100%;  
  height:400px;
  top: 60%;
  -ms-transform: translateY(-50%);
  transform: translateY(-50%);
  
}

.App-header {
  margin: 0;
  position: absolute;
  background: rgb(0, 0, 0); /* Fallback color */
  background: rgba(0, 0, 0, 0.7); /* Black background with 0.5 opacity */
  height: 200px;
  color: #f1f1f1;
  width: 100%;
  top: 60%;
  -ms-transform: translateY(-50%);
  transform: translateY(-50%);
  
}

.App-title {
  margin:0;
  position: relative;  
  text-align: center;
  font-size: 2.5em;
  font-weight: 300;
  font-family: New Century Schoolbook, TeX Gyre Schola, serif;
  top: 50%;
  -ms-transform: translateY(-50%);
  transform: translateY(-50%);
}
