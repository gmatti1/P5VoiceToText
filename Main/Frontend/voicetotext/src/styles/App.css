.Firstpage {
 height: 102vh;
}

.Navbarsize {
  padding :13px;

 }

#navbg {
background-color:#004466;
border-radius: 6px;
}

.App-logobig {
  height: 100px;
  justify-content: center;
}

.App-baby {
  height: 100px;
  justify-content: center;
}

.App-bg {
  margin: 0;
  position: absolute;
  background-image: url(./../assets/medical-wallpaper.jpg);
  opacity: 0.95;
  width: 100%;
  height: 400px;
  top: 56%;
  -ms-transform: translateY(-50%);
  transform: translateY(-50%);
}

.App-title {
  font-size: 1.6em;
}

.Appheaderfp {
  margin: 0;
  position: absolute;
  background: rgb(0, 68, 102); /* Fallback color */
  background: rgba(0, 68, 102, 0.75); /* Black background with 0.5     opacity */
  height: 150px;
  color: #f1f1f1;
  width: 100%;
  top: 56%;
 -ms-transform: translateY(-50%);
  transform: translateY(-50%);
  
}

.App-footer {
  
  position: absolute;
  padding :30px;
  height: 80px;
  width: 100%;
  top: 88.8%;
}

.Arrow {
  position: absolute;
  height: 78%;
  width: 20%;  
  left: 40%;
  top:3%;
}

.Arrow:hover 
{
  opacity: 0.8;
}

.Apptitlefp {
  margin: 0;
  position: relative;
  text-align: center;
  font-size: 2.5em;
  font-family: Josefin Sans;
  top: 50%;
  -ms-transform: translateY(-50%);
  transform: translateY(-50%);
}

.Secondpage {
  height: 100vh;
  background-color: #004466;
  font-color: white;
  font-size: 2.5em;
}

.Uploadheader{
  font-size: 0.7em;
  color:white;
  font-family: Josefin Sans;
  margin: 0;
  position: absolute;
  top: 107%; 
  left:33%;
  right:30%

}

.Input{
  font-size: 0.5em;
  background-color: white;   
  border: 2px solid #32CD32;
  font-family: Josefin Sans;
  position: absolute;
  left: 27%;
  top: 116%;
  height: 6.5%;
  width:36%;
  border-radius: 5px;
}

.Buttonformat{
  font-size: 0.5em;
  background-color: white;  
  border: 2px solid #32CD32;
  font-family: Josefin Sans;
  position: absolute;
  right: 27%;
  top: 116%;
  height: 6.5%;
  width: 10%;
  border-radius: 5px;
}

.Buttonformat:hover {
  background-color: #32CD32;
  color: white;
}

.Textdata{

  position: absolute;
  left: 5%;
  top: 128%;
  border: 2px solid #32CD32;
  border-radius: 5px;
  Height:71.7%;
  Width: 43%;
}

.LabelTextdata {
font-family: Josefin Sans;
font-size: 0.6em;
color:white;

}

#Textarea {

font-family: Josefin Sans;
font-size: 0.43em;
background-color: #e6faff;
height:310px;
}

.Textareasize{
position: absolute;
width:100%;
height:310px;
}


.Save{
  font-size: 0.4em;
  background-color: #e6faff;  
  border: 2px solid #32CD32;
  font-family: Josefin Sans;
  position: absolute;
  left:0%;
  top: 87%;
  border-radius: 5px;
  color:black;
  width:100%;
  height:8%;

}

.Save:hover {
  background-color: #32CD32;
  
}

.Categorydata{

  position: absolute;
  right: 5%;
  top: 128%;
  border: 2px solid #32CD32;
  border-radius: 5px;
  Height:71.7%;
  Width: 43%;
}

.Tablesize{
position: absolute;
width:100%;
height:340px;
}

#TableText{
font-family: Josefin Sans;
font-size: 0.41em;
background-color: #e6faff;

}

/* width scrollbar */
::-webkit-scrollbar {
 height:7px;
width:7px;
}

/* Track */
::-webkit-scrollbar-track {
  box-shadow: inset 0 0 5px grey; 
  border-radius: 15px;
}
 
/* Handle */
::-webkit-scrollbar-thumb {
  background:  #009900; 
  border-radius: 10px;
}

/* Handle on hover */
::-webkit-scrollbar-thumb:hover {
  background:  #006600; 
}