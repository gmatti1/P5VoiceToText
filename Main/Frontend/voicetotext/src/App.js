import React, { Component } from 'react';
import './App.css';
import Logo from './pchlogo.png';
import 'bootstrap/dist/css/bootstrap.min.css';
import { BrowserRouter as Router, Route, Link } from 'react-router-dom';
import Navbar from './Navbar';
import Apppage2 from './Apppage2';

class App extends Component {
  render() {
    return (
	<div className="App">
	<Router>
			<Route path={"apppage2"} component={Apppage2} />
	
       
			<Navbar /> 
			<div className="App-bg" />
			<header className="App-header">
				<h1 className="App-title">
					<Link to="apppage2">Voice-To-Text Application of Pediatric IMIST-AMBO Criteria
					</Link>
				</h1>
			</header>
			
		
      </Router>		
	</div>
    );
  }
}

export default App;

