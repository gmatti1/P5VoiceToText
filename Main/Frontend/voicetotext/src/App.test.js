import React from 'react';
import {render, cleanup} from '@testing-library/react';
import App from './App';

/**
 * This is the test file for App component. 
 *
 * @version 1.0
 * @author [Shashidhar Reddy Vanteru] <svanter1@asu.edu>
 * @copyright [Copyright 2020, P5VoiceToText] (https://github.com/gmatti1/P5VoiceToText)
 *
 */

afterEach(cleanup)
it('should take a snapshot', () => {
	const { asFragment } = render(<App />)
	expect(asFragment(<App />)).toMatchSnapshot()
});

