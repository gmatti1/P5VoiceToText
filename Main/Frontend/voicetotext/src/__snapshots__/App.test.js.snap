// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <div
      class="Firstpage"
    >
      <div
        class="Navbarsize"
      >
        <nav
          class="navbar sticky-top justify-content-center"
          id="navbg"
        >
          <img
            alt="Website logo"
            class="App-logobig"
            src="pchlogo.png"
          />
        </nav>
      </div>
      <div
        class="App-bg"
      />
      <header
        class="Appheaderfp"
      >
        <h1
          class="Apptitlefp"
        >
          Voice-To-Text Application of Pediatric IMIST-AMBO Criteria
        </h1>
      </header>
      <header
        class="App-footer"
      >
        <a>
          <button>
            <img
              alt="arrow"
              class="Arrow"
              src="arrow.png"
            />
          </button>
        </a>
      </header>
    </div>
    <div
      class="Secondpage"
      id="main"
    >
      <div>
        <form>
          <h1
            class="Uploadheader"
          >
            Please upload the audio file
          </h1>
          <input
            class="Input"
            name="voiceFile"
            required=""
            type="file"
          />
          <button
            class="Buttonformat"
            type="submit"
          >
            Upload
          </button>
        </form>
        <div
          class="ORtext"
        >
          or
        </div>
        <a>
          <button
            class="Historybutton"
            type="submit"
          >
            Go To History
            <span
              class="tooltiptext"
            >
              Previously uploaded files
            </span>
          </button>
        </a>
        <div
          class="Textdata"
        >
          <label
            class="LabelTextdata"
          >
            Converted Text
          </label>
          <div />
          <label
            class="Confi"
          >
            Text Confidence : NaN
          </label>
        </div>
        <div
          class="Categorydata"
        >
          <label
            class="LabelTextdata"
          >
            Categorized Text
          </label>
          <div />
        </div>
        <button
          class="btn btn-danger"
          id="calltopopup"
        >
          Add keyword for IMIST-AMBO
        </button>
      </div>
    </div>
    <div
      class="Thirdpage"
      id="hist"
    >
      <div
        class="HistoryComponent"
      >
        <form>
          <div
            class="Historytext"
          >
            History
          </div>
          <div
            class="Historylist"
          >
            <select
              class="Historyselect"
              required=""
              size="10"
            />
          </div>
          <div
            class="SearchHist"
          >
            <input
              class="Search"
              placeholder="ï€‚ Search filename.."
              type="text"
              value=""
            />
          </div>
        </form>
        <button
          class="btn btn-danger center"
          id="histcalltopopup"
        >
          See saved result
        </button>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
